prow/spyglass/artifacts.go

```diff
@@ -55,11 +55,7 @@ func (s *Spyglass) ListArtifacts(ctx context.Context, src string) ([]string, err
 	// Don't care errors that are not supposed logged as http errors, for example
 	// context cancelled error due to user cancelled request.
 	if err != nil && err != context.Canceled {
-		if config.IsNotAllowedBucketError(err) {
```

Comment:
There's no reason to drop this conditional --- just like line 895 in in deck/main.go, only do the debug log if this condition is satisfied.

Reply:
Restored.